release-process-documentation/release-team-guides/ci-signal-playbook.md

```diff
@@ -1,41 +1,141 @@
 # CI Signal Lead Playbook
 
-Explicit detail is important:
-- If you're looking for answer that's not in this document, don't just ask me, please file an issue so we can keep the document current.
-- If I don't file an issue for a failing job or test, _I will lose track of it_, so I err on the side of filing an issue even if someone told me something in some other medium
-- If a dashboard isn't listed here, or a test isn't on one of the listed dashboards, **_I'm not looking at it_**
+## Overview of CI Signal responsibilities
+
+CI Signal lead assumes the responsibility of the quality gate for the release. This person is responsible for:
+- Monitoring various e2e tests in sig-release dashboards ([master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking), [master-upgrade](https://k8s-testgrid.appspot.com/sig-release-master-upgrade), [x.y-blocking](https://k8s-testgrid.appspot.com/sig-release-1.11-blocking)) on a regular basis and providing early and continuous signals on release and test health to both Release team and SIGs
```

Comment:
I'd like to see wording like "Continuously monitoring various e2e tests across the entire release cycle..."  and somewhere a few sentences also about this being throughout the release cycle and that we're trying to avoid a workflow of:

* a month of destabilizing changes and accepting test failures, followed by a month of re-stabilizing to passing tests

and instead have continuous integration with a fail-fast-fix-fast mentality and the culture being one of tests continuously passing.